packages/types/lib/environment/db.ts

```diff
@@ -9,7 +9,7 @@ export interface EnvironmentVariable extends Timestamps {
     value_tag?: string | null;
 }
 
-export interface Environment extends Timestamps {
+export interface DBEnvironment extends Timestamps {
```

Comment:
It might be too difficult to do right now but I would prefer to see `DB...` types used internally in model where data is written/read from the db and have it mapped/returned as non `DB` types for the rest of the codebase to use. 
Like that we can take care of any necessary type or case transformation. That's one of the reason we have camelCase and snake_case mixed all over the codebase.

Reply:
What would you suggest in term of naming? 
Something like `DBEnvironment` for the model, `Environment` for internal use and `APIEnvironment` for external use? Could work, not entirely sure that's it's worth it. 
Just to clarify my view, for me DB means internal but shorter, that allows us to differentiate two non-compatible interface, but that's about it. 
For the camelCase and snake_case issue that's for sure annoying, but it's a lot of effort for not a lot of benefits (except making sure we only select full model every time). Plus it's totally possible to use camelCase in pg (just a bit annoying to quote fields) so the difference might not even exists for new table if we wanted to.

Happy to hear more possible things that could be solved by having more types, I might forgot something.